src/Xamarin.Android.Build.Tasks/Tasks/AndroidComputeResPaths.cs

```diff
@@ -95,7 +95,7 @@ public override bool RunTask ()
 						if (!string.IsNullOrEmpty (ProjectDir)) {
 							var fullRelPath = Path.GetFullPath (rel).Normalize (NormalizationForm.FormC);
 							var fullProjectPath = Path.GetFullPath (ProjectDir).Normalize (NormalizationForm.FormC);
-							if (fullRelPath.StartsWith (fullProjectPath)) {
+							if (fullRelPath.StartsWith (fullProjectPath, StringComparison.Ordinal)) {
```

Comment:
Paths *generally* should use `StringComparison.OrdinalIgnoreCase`: https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#choosing-a-stringcomparison-member-for-your-method-call

> | Data | Behavior | Corresponding System.StringComparison value |
> | -- | -- | -- |
> | File paths | A non-linguistic identifier, where case is irrelevant; especially data stored in most Windows system services. | OrdinalIgnoreCase |